price-feeder/oracle/provider/osmosis.go

```diff
@@ -46,6 +47,18 @@ type (
 		Close  float64 `json:"close"`
 		Volume float64 `json:"volume"`
 	}
+
+	// OsmosisPairsSummary defines the response structure for an Osmosis pairs
+	// summary.
+	OsmosisPairsSummary struct {
+		Data []OsmosisPairData `json:"data"`
+	}
+
+	// OsmosisPairData defines the data response structure for an Osmosis pair.
+	OsmosisPairData struct {
+		BaseSymbol  string `json:"base_symbol"`
+		QuoteSymbol string `json:"quote_symbol"`
```

Comment:
We should keep this consistent with the convention for `GatePairSummary` : 
```suggestion
		Base  string `json:"base_symbol"`
		Quote string `json:"quote_symbol"`
```